google/ggrc-core
# GGRC-4154 Revert GGRC-3158

# Issue description

This PR should fix the issue described by GGRC-4154 which was caused by https://github.com/google/ggrc-core/pull/6993, PR https://github.com/google/ggrc-core/pull/7081 that also tries to fix GGRC-4154 cannot be merged at this time because of an issue.

# Steps to test the changes

Precondition:

Audit with control snapshot;
Created Assessment template with mandatory LCA (e.g. Dropdown with required comment)
Steps to reproduce:

Generate Assessment based on control snapshot and Assessment template
Navigate to Assessment Info pane and click Complete button
Actual Result:
If Complete Assessment w/o filling out mandatory LCA, blank Info pane is shown and tab is closed
Expected Result:
Info pane should not be blank If Complete Assessment w/o filling out mandatory LCA;
"Add required info" should be shown in red.

# Sanity checklist




# PR Review checklist
